src/js/disability-benefits/526EZ/actions/index.js

```diff
@@ -0,0 +1,64 @@
+import Raven from 'raven-js';
+/* eslint-disable */
+import { apiRequest } from '../../../../platform/utilities/api';
+import { fetchInProgressForm } from '../../../common/schemaform/save-in-progress/actions';
+
+export const ITFStatuses = Object.freeze({
+  active: 'active',
+  claim_received: 'claim_received', // eslint-disable-line camelcase
```

Comment:
When we've done this kind of "constants bucket" object before, we've used camelCase for the property names.